core/io/ip.cpp

```diff
@@ -242,12 +301,13 @@ void IP::get_local_addresses(List<IPAddress> *r_addresses) const {
 		}
 	}
 }
-
 void IP::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("resolve_hostname", "host", "ip_type"), &IP::resolve_hostname, DEFVAL(IP::TYPE_ANY));
+	ClassDB::bind_method(D_METHOD("resolve_hostname_addresses", "host", "ip_type"), &IP::resolve_hostname_addresses);
```

Comment:
You should bind the default value with `DEFVAL(IP::TYPE_ANY)` like for `resolve_hostname`.

Reply:
Noted